coco/endpoint.py

```diff
@@ -251,12 +252,12 @@ def client_call(self, host, port, args):
             Is expected to include all values of the endpoint.
         """
         data = copy(self.values)
-        if data:
+        if data and args:
```

Comment:
The scheduler shouldn't be able to call an endpoint that specifies values,right?
In that case `if data:` should have been enough and maybe the scheduler should make sure that none of its endpoints has `.values`

Reply:
ah, right, I forgot we had discussed that. I'll add that check to the `_gen_timers` method.